src/assets/constants.ts

```diff
@@ -0,0 +1,50 @@
+export const galleryImages: { img: string; id: number }[] = [
```

Comment:
Смущает название. По смыслу это должен быть список проектов.
```suggestion
export const projects: { img: string; id: number }[] = [
```